### PR TITLE
Documentation/zenko 2986 crud replication from s3 api 2nd

### PR DESCRIPTION
**Pull request by wabernat**
Added aws cli s3 api commands plus links to API for create/put bucket replication.

### CODE REVIEW

**Review by rachedbenmustapha — COMMENTED** [review]
The replication policies should have
```
"Role": "arn:aws:iam::root:role/s3-replication-role",
```
as a sibling of `Rules`

File: docs/docsource/operation/XDM_GUI/bucket_operations/update_a_replication_workflow.rst [hunk 1]
```diff
@@ -40,6 +40,7 @@ Where ``configuration`` is a JSON object including the following syntax:
            }
          }
        ]
+     "Role": "arn:aws:iam::root:role/s3-replication-role",
```
Comment (rachedbenmustapha) [inline docs/docsource/operation/XDM_GUI/bucket_operations/update_a_replication_workflow.rst]:
```suggestion
     ],
     "Role": "arn:aws:iam::root:role/s3-replication-role",  
```
A `,` is needed between the values to be a valid JSON snippet (applies to Create as well)

File: docs/docsource/operation/XDM_GUI/bucket_operations/delete_a_replication_workflow.rst [hunk 2]
```diff
@@ -14,3 +14,13 @@ To delete a replication workflow rule:
 #. The workflow is removed from the menu. If no workflows remain, you are
    prompted to create a new worflow.
 
+Delete a Replication Workflow from the Command Line
+---------------------------------------------------
```
Comment (cathydossantospinto) [inline docs/docsource/operation/XDM_GUI/bucket_operations/delete_a_replication_workflow.rst]: I'd say "Deleting a Replication Workflow from the Command Line" to keep a consistency with the other two operations "Creating" & "Updating" 😉

File: docs/docsource/operation/XDM_GUI/bucket_operations/update_a_replication_workflow.rst [hunk 3]
```diff
@@ -15,3 +15,32 @@ Once you've created a replication workflow, you can update it as follows:
    active state of the workflow.
 
 #. When you have made all desired changes, click **Save Changes**.
+
+Updating a Replication Workflow Using the AWS CLI
+-------------------------------------------------
+
+Updating a replication workflow is the same logic as :ref:`creating one<Create
```
Comment (cathydossantospinto) [inline docs/docsource/operation/XDM_GUI/bucket_operations/update_a_replication_workflow.rst]:
```suggestion
Updating a replication workflow follows the same logic as :ref:`creating one<Create
```

**Comment by wabernat** [comment]
@RachedBenMustapha, your approval would make my life complete.

**Review by rachedbenmustapha — APPROVED** [review]

**Review by saitharunthodupunuru — APPROVED** [review]

**Pull request merged**